cmd/cluster.go

```diff
@@ -77,6 +77,10 @@ func ClusterUp(clusterFile, authType string) (string, string, string, string, er
 		return ApiURL, caCrt, clientCert, clientKey, err
 	}
 
+	err = kubeCluster.DeployNetworkPlugin()
```

Comment:
Should be generic call to deploy the addons here. Then inside the addons deploy code you can distinguish between the plugins types.

Reply:
I added env variable configuration. network plugin name will be passed in the cluster yaml file and we can add env variables add more plugins. We will also update this as we add plugins if needed.